assets/dev/js/editor/elements/views/container.js

```diff
@@ -412,6 +412,12 @@ const ContainerView = BaseElementView.extend( {
 				this.reInitEmptyView();
 			}
 
+			// Todo: remove in future version.
+			// Remove together with support for physical properties inside the Mega Menu & Nested Carousel widgets.
+			if ( ! this.model.get( 'isInner' ) ) {
+				this.$el[ 0 ].dataset.coreV316Plus = 'true';
+			}
+
```

Comment:
I have added an attribute on the parent container so that we can more easily distinguish between the old and the new version. I need that distinction inside the Nested Widgets in pro.